server/model.js

```diff
@@ -1,19 +1,21 @@
 /* eslint-disable linebreak-style */
-const mysql = require('mysql');
-const config = require('../configDB.js')
-const connection = mysql.createConnection(config)
 
+const pg = require('pg');
+const config = require('../configDB.js');      
+const client = new pg.Client(config);
+
+client.connect();
```

Comment:
You might want to add an error handler, just in case your connection doesn't go through!